.github/buildomat/jobs/build-and-test-linux.sh

```diff
@@ -29,7 +29,7 @@ mkdir "$TEST_TMPDIR"
 # Put "./cockroachdb/bin" and "./clickhouse" on the PATH for the test
 # suite.
 #
-export PATH="$PATH:$PWD/out/cockroachdb/bin:$PWD/out/clickhouse"
+export PATH="$PATH:$PWD/out/cockroachdb/bin:$PWD/out/clickhouse:$PWD/out/dendrite-stub/bin"
```

Comment:
This new path should be added to `tools/setup-path.sh`. We should probably also document these environment variable requirements in `docs/how-to-run.adoc`.